.github/workflows/publish.yaml

```diff
@@ -29,6 +29,9 @@ jobs:
     permissions:
       id-token: write
       contents: read
+    strategy:
+      matrix:
+        python-version: [ "3.9", "3.10" ]
```

Comment:
Is this common practice to upload multiple times to pypi with the same version number but different versions of python used to build?

Reply:
I think multiple Python versions can be associated with one version number, as long as they are packaged as separate wheels or distributions that specify compatibility for different Python versions.